packages/ui/src/components/editor/panels/Viewport/tools/SceneHelpersTool.tsx

```diff
@@ -0,0 +1,94 @@
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import { downloadScreenshot } from '@etherealengine/editor/src/functions/takeScreenshot'
+import { EditorHelperState, PlacementMode } from '@etherealengine/editor/src/services/EditorHelperState'
+import { useMutableState } from '@etherealengine/hyperflux'
+import { RendererState } from '@etherealengine/spatial/src/renderer/RendererState'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { LuMousePointerClick, LuMove3D } from 'react-icons/lu'
+import { twMerge } from 'tailwind-merge'
+import Button from '../../../../../primitives/tailwind/Button'
+import ColliderAtomsSvg from './icons/collider_atoms.svg?react'
+import RulerSvg from './icons/ruler.svg?react'
+import ScreenshotSvg from './icons/screenshot.svg?react'
+
+export default function SceneHelpersTool() {
+  const { t } = useTranslation()
+  const editorHelperState = useMutableState(EditorHelperState)
+  const rendererState = useMutableState(RendererState)
+
+  return (
+    <div className="flex items-center gap-1">
+      <Button
+        startIcon={<LuMousePointerClick />}
+        onClick={() => editorHelperState.placementMode.set(PlacementMode.CLICK)}
+        variant={editorHelperState.placementMode.value === PlacementMode.CLICK ? 'outline' : 'transparent'}
+        title={t('editor:toolbar.placement.click')}
+        className="px-0"
+      />
+      <Button
+        startIcon={<LuMove3D />}
+        onClick={() => editorHelperState.placementMode.set(PlacementMode.DRAG)}
+        variant={editorHelperState.placementMode.value === PlacementMode.DRAG ? 'outline' : 'transparent'}
+        title={t('editor:toolbar.placement.drag')}
+        className="px-0"
+      />
+      {/* <Tooltip
+        title={t('editor:toolbar.helpersToggle.lbl-helpers') + ' | ' + t('editor:toolbar.helpersToggle.info-helpers')}
+        > */}
+      <Button
+        startIcon={<RulerSvg />}
+        onClick={() => rendererState.physicsDebug.set(!rendererState.physicsDebug.value)}
+        variant={rendererState.physicsDebug.value ? 'outline' : 'transparent'}
+        title={t('editor:toolbar.helpersToggle.lbl-helpers')}
+        className={twMerge('px-0', rendererState.physicsDebug.value && 'border border-solid border-transparent')}
+      />
+      {/* </Tooltip> */}
```

Comment:
Why have we commented out the tool tips?

Reply:
Tooltips are not positioning themselves correctly. I am working on a fix of the popover to be more dynamic in width and height.
The design team is using a different set of icons altogether which are completely absent in react-icons - we have to change all the icons later. Using svgs now as a temporary solution.